packages/backend/hardhat.config.js

```diff
@@ -0,0 +1,33 @@
+require('@nomicfoundation/hardhat-toolbox')
+
+module.exports = {
+  defaultNetwork: 'arb',
+  networks: {
+    hardhat: {
+      blockGasLimit: 12000000,
+      chainId: 421613, // match arb goerli
+    },
+    local: {
+      url: 'http://localhost:8545',
+      blockGasLimit: 12000000,
+      chainId: 421613, // match arb goerli
+      accounts: [
+        '0xbca43b51928100e446f2265f1493c3e9187d62eb26ef043f8ecbae6cb27e4f51',
+      ],
+    },
```

Comment:
this is confusing
the hardhat network is where we run `npx hardhat test`
and local is where we run `npx hardhat node` (it will run on http://localhost:8545)
so I think it doesn't need a chainId

Reply:
removed